demos/python_demos/clip_image_classification/README.md

```diff
@@ -15,19 +15,22 @@ The picture below shows the execution flow in the graph.
 ```bash
 git clone https://github.com/openvinotoolkit/model_server.git
 cd model_server
-make python_image
+make python_image RUN_TESTS=0
```

Comment:
RUN_TESTS is no longer needed since we changed Makefile to use public image instead of building.